libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/MapboxNativeNavigator.kt

```diff
@@ -34,6 +34,13 @@ interface MapboxNativeNavigator {
         tilesConfig: TilesConfig
     ): MapboxNativeNavigator
 
+    /**
+     * Reset the navigator state with the same configuration. The location becomes unknown,
+     * but the [NavigatorConfig] stays the same. This can be used to transport the
```

Comment:
Should we also some of this to `MapboxNavigation#resetTripSession` KDocs?

Reply:
Yeah, I meant adding some clarifications here https://github.com/mapbox/mapbox-navigation-android/blob/d00ec6150cde9352c7b352d068b204dd89c0cefb/libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt#L265-L267 For example `This can be used to transport the...`

Reply:
OH yeah I misinterpreted. Thanks for clarifying 😅